drivers/sensor/mma8451q/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+zephyr_library()
```

Comment:
Please add a copyright/license header to every new file

Reply:
ok - only saw in a few (I'll add though)

Reply:
We're usually pretty good about this in .c/.h files, but not as good in others.